.env.example

```diff
@@ -3,4 +3,7 @@ ETHEREUM_NETWORK =  ""
 INFURA_API_KEY = ""
 VOTE_ID= ""
 CHAIN_ID=1
-VOTING_CONTRACT_ADDRESS = ""
\ No newline at end of file
+VOTING_CONTRACT_ADDRESS = ""
+MAINNET_DEPLOYER_PRIVATE_KEY = ""
```

Comment:
I noticed there are some env variables being used in the script that are not included in this example. Would you mind going through the script and including all necessary env variables in this example file?